### ISSUE TITLE
Group fields - unable to sort/drag & unable to remove/delete the 1st item

### ISSUE THREAD

**Issue by paolosax** [CMB2/CMB2#1483]
## Describe the bug
I'm having issues on repeater/group fields, as now it seems that I can't sort/order each single fields groups, and also now I cannot remove the 1st group item

## Steps to reproduce (I have confirmed I can reproduce this issue on the [`develop`](https://github.com/CMB2/CMB2/tree/develop) branch):
--
## Possible Solution
--
### Possible Solution's Risk Level
--
## Screenshots
--

## CMB2 Field Registration Code:

```php
$features = new_cmb2_box(array(
    'id'                => PREFIX . 'features',
    'title'             => esc_html__('Features section', 'my-slug'),
    'object_types'      => array('my-cpt-slug'),
    'show_in_rest'      => true,
    'context'           => 'normal',
    'priority'          => 'high',
    'closed'            => true,
));
$cards = $features ->add_field(array(
    'id'          => PREFIX . 'features_cards',
    'type'        => 'group',
    'repeatable'  => true, // use false if you want non-repeatable group
    'options'     => array(
        'group_title'       => __('Card {#}', 'my-slug'), // since version 1.1.4, {#} gets replaced by row number
        'add_button'        => __('Add another feature', 'my-slug'),
        'remove_button'     => __('Remove feature', my-slug'),
        'sortable'          => true,
        'closed'         => true, // true to have the groups closed by default
        'remove_confirm' => esc_html__('Are you sure you want to remove?', 'my-slug'), // Performs confirmation before removing group.
    ),
));
$features ->add_group_field($cards , array(
    'id'   => PREFIX . 'features_card_image',
    'name' => esc_html__('Image', 'my-slug'),
    'desc' => '',
    'type' => 'file_list',
    'preview_size' => array(160, 90), // Default: array( 50, 50 )
    'query_args' => array('type' => 'image'), // Only images attachment
    'options' => array(
         'url' => false, // Hide the text input for the url
     ),
));

[other few text fields]
```

## Your Environment

Browser name and version: Chrome
Operating System and version (desktop or mobile): Windows11

**Comment by tw2113** [comment]
Can you open your browser's developer console and observe if there's any errors occurring at the javascript level?

Also is this with classic or block editor where you're seeing it?

**Comment by paolosax** [comment]
Hello Michael - nope, no js errors in console (except some jquery migrate warnings). 
And yes, block editor is enabled...

**Comment by tw2113** [comment]
Will try to test/confirm soon. I believe the inability to remove ALL is sort of intentional, even if empty. However I could be wrong with intent there.

**Comment by paolosax** [comment]
As far as I know (ie: at least for what I learned using CMB2 quite often), the Group field will always have at least on item, but when you try to delete it it just delete the content of its inner fields... and that's fine. But what I experience now is that is not possible to remove it at all (meaning, remove the content of a single Group field). Weird...

**Comment by tw2113** [comment]
I know in the original issue there's a lot of the boilerplate GitHub issue content still in place, but I want to confirm if you have actually tested with the `develop` branch from GH here? Asking because I believe https://github.com/CMB2/CMB2/commit/920200a948e3b1b2a7ccd65f7a2646a500b09cc6 may fix this as it was a very similar issue.

**Comment by paolosax** [comment]
Yep, it seems that this solve the "1st item delete" issue. 
What I miss now is the "sortable" feature that still doesn't work. 
But now I've the doubt that I'm not working using the right CMB2 version. 
I mean: my Composer require is "cmb2/cmb2": "^2.10.1" (not "dev-develop")

**Comment by tw2113** [comment]
Correct, that require statement is based on the tagged versions and 2.10.1 is our most recent tag. Would not be pulling in changes in the `develop` branch. Not sure how to potentially do that via composer.
